File: python/exe_133.py
A, B, C = list(map(int, input().split()))

if (A > B or A > C):
    if (B < C):
        temp = A
        A = B
        B = temp
    else:
        temp = A
        A = C
        C = temp
if (B > C):
    temp = B
    B = C 
    C = temp
print("{}, {}, {}".format(A, B, C))
    

File: c/exe_133.c
#include <stdio.h>


int	main()
{
	int L, M, N, temp;
	
	scanf("%d%d%d", &L, &M, &N);
	if (L > M || L > N)
	{
		if (M < N)
		{
			temp = L;
			L = M;
			M = temp;
		}else
		{
			temp = L;
			L = N;
			N = temp;
		}
	}if (M > N)
	{
		temp = M;
		M = N;
		N = temp;
	}
	printf("\n %d, %d, %d \n", L, M, N);
	return 0;
}

